programmers/비밀지도.js

```diff
@@ -0,0 +1,24 @@
+function solution(n, arr1, arr2) {
+    
+    let result = [];
+    const output = [];
```

Comment:
```suggestion
    const output = [];
    let result = [];
```
보통 const, let 이 모두 있는 경우에는 const를 상단에 적어주는게 좋다고합니다.